packages/migrations/src/clickhouse-actions/011-audit-logs.ts

```diff
@@ -0,0 +1,20 @@
+import type { Action } from '../clickhouse';
+
+export const action: Action = async exec => {
+  await exec(`
+    CREATE TABLE IF NOT EXISTS "audit_log"
```

Comment:
Do we need to configure indexing?

@TuvalSimha please confirm what queries are you making here, and @kamilkisiela can help with the indexes.

Reply:
reminder of this one